roles/lib_openshift/src/class/oc_process.py

```diff
@@ -175,11 +175,11 @@ def run_ansible(params, check_mode):
                 update = True
 
         if not update:
-            return {"changed": update, "results": api_rval, "state": "present"}
+            return {"changed": update, "results": api_rval, "state": state}
 
         for cmd in rval:
             if cmd['returncode'] != 0:
-                return {"failed": True, "changed": update, "results": rval, "state": "present"}
+                return {"failed": True, "changed": update, "msg": rval, "state": state}
```

Comment:
I see the subject is
> Fixed a bug when oc_process executes fail_json

and the commit message says
> Fixed a bug. Ansible requires a msg param when module.fail_json.

But the code changes only change the value of the `state` return parameter from strings to what the module `state` parameter was set to when the module is invoked.

Looks like the `msg` param is only added in one return statement.

So the other changes are just regular bug fixes (return the actual requested state, not a hard-coded string) and the change in `src/class/oc_process.py` on line ~182 adds the actual `msg` param.

If that's all correct and intended then +1.